osmosis-labs/osmosis-frontend
# Publish Stage

* Settings modal closes #1142 
* Progress on headless #1148 
* Fix: Closes #1255 
* Fix: Closes #1260
* Fix: Closes #1280 
* Fix: transfer status not updating
* Update inlang config (translations frontend)
* Filters completed gauges
* Reorder assets
* ICNS names in transfer modal
* Warning when adding liquidity to pool with scaling factor controller

Test Guide
* Settings Modal
* Mobile Profile Modal
* Swap tool
  * Large amounts
  * Token pairs that don't have routes (see non-routes here: https://github.com/osmosis-labs/osmosis-frontend/issues/1231)
* Make sure no pages fail with changes to gauge query (assets page)
* Transfer both assets via IBC and Axelar. To test failure for Axelar, pay with insufficient fee. Transfers should not remain pending in the table.
  * Notice ICNS names for both osmosis and counterparty chains (cosmos, stars)
